helm/helm-app-database/src/test/resources/test.properties

```diff
@@ -0,0 +1,2 @@
+ts.app.log.enable=false
+ts.global.generated-service.enabled=false # generated-service should be set to False on Helm scenarios
```

Comment:
There isn't too much information in that comment... I suppose it should be set to `false` as you set it to `false`, but maybe you can give a hint why (or if it's so clear, remove comment; not clear to me till I read more code)

Reply:
Code comment updated!